src/commands/call.js

```diff
@@ -38,13 +41,23 @@ class CallCommand extends ApifyCommand {
             if (err.type === 'record-not-found') throw new Error(`Act ${runOpts.actId} not found!`);
             else throw err;
         }
-        console.dir(run);
 
-        const log = await apifyClient.logs.getLog({ logId: run.id });
-        console.log(log);
+        outputs.link('Act run detail', `https://my.apify.com/acts/${run.actId}#/runs/${run.id}`);
+
+        const streamRequest = http.get(`http://api.apify.com/v2/logs/${run.id}?stream=1`)
```

Comment:
Can we use `https`?

And:

* You should be also listening for `streamRequest.on('error', ...`. What happens now in a case of an error?
* You don't need `waitForTaskFinish` - just listen for `streamRequest.on('end',...`

Reply:
_You don't need waitForTaskFinish - just listen for streamRequest.on('end',..._

I need to wait until waitForFinish is reach and then get build as well. Is it nessesary to rewriting it?